static/src/javascripts/projects/facia/modules/ui/lazy-load-containers.js

```diff
@@ -11,28 +12,50 @@ define([
     qwery,
     _,
     detect,
+    storage,
     mediator
 ) {
-    var distanceBeforeLoad = detect.getViewport().height;
+    var storageKey = 'gu.front.lazystate',
+        distanceBeforeLoad = detect.getViewport().height;
+
+    function saveState(numOpenContainers) {
+        return storage.session.set(storageKey, {
+            pathname: window.location.pathname,
+            openContainers: numOpenContainers
+        });
+    }
+
+    function isHidden(el) {
+        return el.offsetParent === null;
```

Comment:
As per our discussion, this triggers layout, so should be in a `fastdom.read` if you want to use it. 

http://gent.ilcore.com/2011/03/how-not-to-trigger-layout-in-webkit.html